### ISSUE TITLE
Determine solution for a11y inspector scrolling elements to the bottom right in Storybook

### ISSUE THREAD

**Issue by kevinparkerson** [salesforce/design-system-react#1675]
When working with components that have scrollable containers in Storybook, the a11y inspector causes odd bugs due to it checking all elements for accessibility. This can cause rendering issues or other strange behaviors that are difficult to diagnose during component development, and in the worst case, convince the developer the component is to blame and cause them to write an unnecessary workaround. The details of why it occurs can be viewed here:

https://github.com/storybooks/storybook/issues/4274

Due to this, the a11y inspector has been temporarily disabled, but if a proper solution for this problem can be determined it should be reenabled in the future as it was a helpful tool.

**Comment by interactivellama** [comment]
Closing since the a11y addon was removed and aXe testing is being done through Storyshots now in `npm test:accessibility`.
